### PR TITLE
Migrate company 'PX'

### PR DESCRIPTION
**Pull request by swingerman**
Adds or updates the following companies, events, or universities:
 - 
 - 

#### Company updates
 - [x] I have put the most recent relevant date in the "Last Update" column
 - [ ] I have linked to the article about the change, not the company's homepage (Please put N/A if there is no public post)

#### Event updates
 - [ ] I have linked to the article about the change, not the event's homepage

#### University updates
 - [ ] I have linked to the article about the change, not the university's homepage

### CODE REVIEW

File: _data/companies/px.yaml [hunk 1]
```diff
@@ -0,0 +1,7 @@
+---
+name: PX
```
Comment (drewbrew) [inline _data/companies/px.yaml]: What does PX stand for?

Comment (swingerman) [reply _data/companies/px.yaml]: PX = Private Exchange, PX platform, PX inc.

Comment (drewbrew) [reply _data/companies/px.yaml]:
```suggestion
name: PX (Private Exchange)
```

What do you think about this change?

Comment (swingerman) [reply _data/companies/px.yaml]: I would prefer 'PX Platform'

**Pull request merged**